### PR TITLE
Rework broadcast message descriptions

### PR DESCRIPTION
**Pull request by noon-dawg**
## Summary

This pull request reworks the broadcast message descriptions, and related files, to conform to the [writing style](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md).

## Links

N/A

## Checklist

- [x] Writing style
- [x] API design standards

### CODE REVIEW

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 1]
```diff
@@ -115,29 +119,30 @@ properties:
...
     type: string
     format: date-time
-    description: The split test start time.
+    description: |-
+      Date and time at which the split test messages are schedule to send.
   startSendingTime:
     type: string
     format: date-time
-    description: The start sending time.
+    description: Date and time at which the message is schedule to send.
```
Comment (vykhrystiuk) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:127]:
```suggestion
    description: Date and time at which the message is scheduled to send.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 2]
```diff
@@ -4,104 +4,108 @@ required:
...
-    description: >
-      The filter that will be used on customer selection during the broadcast
-      message processing.
+    description: |-
+      Use this filter to select customers during broadcast message
+      processing.
 
-      The syntax is the same as the `query` filter, without `filter=` prefix.
+      This field uses the same syntax as the `query` filter, but without
+      the `filter=` prefix. Examples: `firstName:John`, `firstName:John;lastName:Doe`.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:18]: https://sembr.org/
```suggestion
      This field uses the same syntax as the `query` filter,
      but without the `filter=` prefix. Examples: `firstName:John`, `firstName:John;lastName:Doe`.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 3]
```diff
@@ -4,104 +4,108 @@ required:
...
 
-      For example: `firstName:John`, `firstName:John;lastName:Doe`, etc.
-
-      Attention: You should not URL encode the filter value.
+      > **Important**: Do not URL encode the filter value.
     example: 'firstName:John;lastName:Doe'
   title:
     type: string
-    description: The title of the messages (this title will appear in reports).
+    description: Title of the message. This title is displayed in reports.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:24]:
```suggestion
    description: Title of the message. This title displays in reports.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 4]
```diff
@@ -4,104 +4,108 @@ required:
...
+          description: ID of the message.
         version:
           type: string
-          description: >-
-            The message version (by default is empty, but if you make multiple
-            versions, you can name or number them to distinguish them).
+          description: |-
+            Version number of the message. Use this field to distinguish
+            between multiple messages by name and version number. If
+            there are no versions, this field is empty.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:41]:
```suggestion
            Version number of the message.
            Use this field to distinguish between multiple messages by name and version number.
            If there are no versions, this field is empty.
```
https://sembr.org/

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 5]
```diff
@@ -4,104 +4,108 @@ required:
...
         weight:
           type: integer
-          description: >
-            The value for random-weighted picking of a template in the case of a
-            split test.
+          description: |-
+            Weight distribution value that is assigned to a template
+            for a split test. Each template in a split test can be
+            assigned a weight. The higher the weight value, the more likely
+            the message template will be used.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:48]:
```suggestion
            Weight distribution value that is assigned to a template for a split test.
            Each template in a split test can be assigned a weight.
            The higher the weight value, the more likely the message template will be used.
```
https://sembr.org/

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 6]
```diff
@@ -4,104 +4,108 @@ required:
...
 
-            The split test algorithm does not factor localization when making a
-            weighted-random template selection.  Therefore, a version will be
-            selected first, and then after a version is selected a specific
-            localization will be selected.  Take the case where two versions
-            have different locale content -- the locales of the content is not
-            considered when selecting the version.
+            The split test algorithm does not assess locale when
+            making a weighted template selection.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:51]:
```suggestion
            The split test algorithm does not assess locale when making a weighted template selection.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 7]
```diff
@@ -4,104 +4,108 @@ required:
...
-            The split test algorithm does not factor localization when making a
-            weighted-random template selection.  Therefore, a version will be
-            selected first, and then after a version is selected a specific
-            localization will be selected.  Take the case where two versions
-            have different locale content -- the locales of the content is not
-            considered when selecting the version.
+            Use split tests to determine which content is most effective.
+            Split tests send variations of the same message to different
+            customers or leads. You can then send the winning message
+            to the remaining customers or leads.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:63]:
```suggestion
            Split tests send variations of the same message to different customers or leads.
            You can then send the winning message to the remaining customers or leads.
```
https://sembr.org/

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 8]
```diff
@@ -4,104 +4,108 @@ required:
...
-                  The sender address. It must be a **verified** address. # todo
-                  add a link to help docs on how to verify an email address.
+                description: |-
+                  Email address of the sender.
+
+                  > **Important:** This value must be a verified email address.
+
+                  To verify an email address:
+                    1. [Create an email delivery setting](../../Email-delivery-settings/operation/PostEmailDeliverySetting).
+                    In the response, you receive the email and a token.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:78]:
```suggestion
                      In the response, you receive the email and a token.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 9]
```diff
@@ -4,104 +4,108 @@ required:
...
+                    1. [Verify the email delivery](../../Email-delivery-settings/operation/PutEmailDeliverySettingsVerification) by passing the
+                    token as the path parameter.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:80]:
```suggestion
                    1. [Verify the email delivery](../../Email-delivery-settings/operation/PutEmailDeliverySettingsVerification) by passing the token as the path parameter.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 10]
```diff
@@ -4,104 +4,108 @@ required:
...
                 maxLength: 254
                 example: example@example.com
               subject:
                 type: string
-                description: The message subject. The template placeholders are allowed.
+                description: |-
+                  Subject of the message. The use of template placeholders
+                  is permitted for this field.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:87]:
```suggestion
                description: |-
                  Subject of the message.
                  The use of template placeholders is permitted for this field.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 11]
```diff
@@ -4,104 +4,108 @@ required:
...
               text:
                 type: string
-                description: |
-                  The message's text body.
-                  Leave empty to use content from "html".
-                  The template placeholders are allowed.
+                description: |-
+                  Text body of the message. To use content from the
+                  `html` field, leave this field empty. The use of
+                  template placeholders is permitted for this field.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:95]:
```suggestion
                  Text body of the message.
                  To use content from the `html` field, leave this field empty.
                  The use of template placeholders is permitted for this field.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 12]
```diff
@@ -4,104 +4,108 @@ required:
...
               html:
                 type: string
-                description: |
-                  The message's html body.
-                  Leave empty to use content from "text".
-                  The template placeholders are allowed.
+                description: |-
+                  HTML body of the message. To use content from the
+                  `text` field, leave this field empty. The use of
+                  template placeholders is permitted for this field.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:102]:
```suggestion
                  HTML body of the message.
                  To use content from the `text` field, leave this field empty.
                  The use of template placeholders is permitted for this field.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 13]
```diff
@@ -4,104 +4,108 @@ required:
...
                 example: <p>Texte de démonstration</p>
               locale:
                 type: string
-                description: >-
-                  The language locale identifier according to [RFC
-                  5646](https://tools.ietf.org/html/rfc5646).
+                description: |-
+                  Language locale identifier in [RFC
+                  5646](https://tools.ietf.org/html/rfc5646) format.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:108]:
```suggestion
                  Language locale identifier in [RFC 5646](https://tools.ietf.org/html/rfc5646) format.
```

**Review by adamaltman — CHANGES_REQUESTED** [review]

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml [hunk 14]
```diff
@@ -4,104 +4,108 @@ required:
...
     allOf:
       - $ref: ../ResourceId.yaml
   filter:
     type: string
-    description: >
-      The filter that will be used on customer selection during the broadcast
-      message processing.
+    description: |-
+      Use this filter to select customers during broadcast message
+      processing.
```
Comment (adamaltman) [inline openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:15]:
```suggestion
      Use this filter to select customers during broadcast message processing.
```

File: openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml (hunk 2 above)
Comment (noon-dawg) [reply openapi/components/schemas/BroadcastMessages/BroadcastMessage.yaml:18]: Thanks for sharing! I wasn't aware of this convention

**Comment by noon-dawg** [comment]
Thanks for the review @adamaltman and @vykhrystiuk!

**Review by vykhrystiuk — APPROVED** [review]

**Review by ashkarpetin — APPROVED** [review]

**Review by adamaltman — APPROVED** [review]

**Pull request merged**